homeassistant/components/vicare/sensor.py

```diff
@@ -836,6 +837,15 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
             "forcedlevelfour",
         ],
     ),
+    ViCareSensorEntityDescription(
+        key="supply_pressure",
+        translation_key="supply_pressure",
+        device_class=SensorDeviceClass.ATMOSPHERIC_PRESSURE,
```

Comment:
Probably this is more suitable:

```suggestion
        device_class=SensorDeviceClass.PRESSURE
```